components/application-authenticators/org.wso2.carbon.identity.application.authenticator.social/src/main/java/org/wso2/carbon/identity/application/authenticator/social/google/GoogleOAuth2Authenticator.java

```diff
@@ -57,33 +57,85 @@
 public class GoogleOAuth2Authenticator extends OpenIDConnectAuthenticator {
 
     private static final long serialVersionUID = -4154255583070524018L;
+    private String tokenEndpoint;
+    private String oAuthEndpoint;
+    private String userInfoEndpoint;
 
     private static Log log = LogFactory.getLog(GoogleOAuth2Authenticator.class);
 
     /**
+     * initiate tokenEndpoint
```

Comment:
Use proper linguistics for comments. Start respective sentences with a capital letter in all similar instances.